Elizabeth_Theme/src/css/layout/_welcome_page.scss

```diff
@@ -0,0 +1,60 @@
+.welcome-container {
+	background-color: #0000; 
+	color: #fff;
+	display: flex;
+	flex-direction: column;
+	height: 100vh;
+	max-width: 100%;
+	width: 100vw;
+}
+
+.welcome-content {
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	height: 100vh;
+	justify-content: center;
+}
+
+.description-container {
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+	text-align: center;
+	width: 40em;
+}
+
+.theme-color-text {
+  	color: $theme-main-color;
+}
+
+.welcome-footer-container {
+	padding-left: 20px;
+	padding-right: 20px;
+	padding-bottom: 75px;
+}
+
+.welcome-footer-content {
+	display: flex;
+	justify-content: space-around;
+	max-width: 100%;
+	width: 100%;
+}
+
+.user-content {
+	background-color: #fff;
+	width: 100%;
+	height: 100%;
+}
+
+@media screen and (max-width:1024px){
```

Comment:
Rather than hardcoding a pixel value, can we leverage the media query break points defined in Liferay instead?